From b5f22f43118c2c18ae0cc6f2532e5485cdc3d131 Mon Sep 17 00:00:00 2001
From: Nigel Croxon <nigel.croxon@hpe.com>
Date: Mon, 30 Nov 2015 08:52:25 -0500
Subject: [PATCH 04/10] From: Julian Andres Klode <jak@debian.org> Subject:
 [PATCH gnu-efi] lib/arm/setjmp.S: Use %function instead of @function

@ is a comment character on ARM, so use % instead.

Nigel adjusted the wordwrap on the copyright header.

Signed-off-by: Julian Andres Klode <jak@degian.org>
Signed-off-by: Nigel Croxon <nigel.croxon@hpe.com>
---
 lib/arm/setjmp.S | 19 ++++++++-----------
 1 file changed, 8 insertions(+), 11 deletions(-)

diff --git a/lib/arm/setjmp.S b/lib/arm/setjmp.S
index 6e3fbf0..bd61a8d 100644
--- a/lib/arm/setjmp.S
+++ b/lib/arm/setjmp.S
@@ -1,21 +1,18 @@
 /*
  * Copyright (c) 2006 - 2008, Intel Corporation. All rights reserved.
- * This program and the accompanying materials are licensed and made 
-available
- * under the terms and conditions of the BSD License which accompanies 
-this
- * distribution.  The full text of the license may be found at
- * http://opensource.org/licenses/bsd-license.php.
+ * This program and the accompanying materials are licensed and made
+ * available under the terms and conditions of the BSD License which
+ * accompanies this distribution.  The full text of the license may
+ * be found at http://opensource.org/licenses/bsd-license.php.
  *
  * THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" 
-BASIS,
- * WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR
- * IMPLIED.
+ * BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER
+ * EXPRESS OR IMPLIED.
  */
 	.text
 	.arm
 	.globl	setjmp
-	.type	setjmp, @function
+	.type	setjmp, %function
 setjmp:
 	mov	r3, r13
 	stmia	r0, {r3-r12,r14}
@@ -23,6 +20,6 @@ setjmp:
 	bx	lr
 
 	.globl	longjmp
-	.type	longjmp, @function
+	.type	longjmp, %function
 longjmp:
 	ldmia	r0, {r3-r12,r14}
-- 
2.5.0

